src/main/java/com/example/userManagement/controller/UserHistoryApi.java

```diff
@@ -0,0 +1,61 @@
+package com.example.userManagement.controller;
+
+import com.example.userManagement.model.GetErrorResponse;
+import com.example.userManagement.model.UserResponse;
+import com.example.userManagement.model.index.UserIndexModel;
+import com.example.userManagement.service.UserServiceES;
+import io.swagger.annotations.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import javax.validation.constraints.*;
+import java.util.List;
+import java.util.UUID;
+
+@javax.annotation.Generated(value = "org.openapitools.codegen.languages.SpringCodegen", date = "2021-05-20T10:30:25.036+05:30[Asia/Kolkata]")
+@Validated
+@Api(value = "userHistory", description = "the userHistory API")
+@RestController
+class UserHistoryApi {
+
+
+    @Autowired
+    private UserServiceES userServiceES;
+
+    /**
+     * GET /userHistory/{userId}/change-log : Get the User change log
+     *
+     * @param userId       userId that needs to be deleted (required)
+     * @param from         Start time of the change log in unix epoch format. (required)
+     * @param timeDuration Number of days for which change log is to be returned. (optional, default to 1)
+     * @param limit        Number of items to return. (optional, default to 50)
+     * @param offset       Number of items to skip from start. (optional, default to 0)
+     * @return This is a user info!. (status code 200)
+     * or Error Occurred (status code 400)
+     * or Error Occurred (status code 401)
+     * or Error Occurred (status code 404)
+     */
+    @ApiOperation(value = "Get the User change log", nickname = "getUserChangeLog", notes = "", response = UserResponse.class, tags = {"Change Log",})
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "This is a user info!.", response = List.class),
+            @ApiResponse(code = 400, message = "Error Occurred", response = GetErrorResponse.class),
+            @ApiResponse(code = 401, message = "Error Occurred", response = GetErrorResponse.class),
+            @ApiResponse(code = 404, message = "Error Occurred", response = GetErrorResponse.class)})
+    @GetMapping(value = "/userHistory/{userId}/change-log")
+    public List<UserIndexModel> getUserChangeLog(
+            @ApiParam(value = "userId that needs to be deleted", required = true) @PathVariable("userId") UUID userId,
+            @NotNull @ApiParam(value = "Start time of the change log in unix epoch format.", required = true)
+            @Valid @RequestParam(value = "from", required = true) long from, @Min(1) @Max(31)
+            @ApiParam(value = "Number of days for which change log is to be returned.", defaultValue = "2")
+            @Valid @RequestParam(value = "toDate", required = false, defaultValue = "2") long toDate, @Min(1)
+            @ApiParam(value = "Number of items to return.", defaultValue = "50")
```

Comment:
Let us pick all the default values from Constants file too.

Reply:
yes moved all the repeated variable to utils.Constant